### PR TITLE
Job results fix

### PR DESCRIPTION
**Pull request by annasirunian**

### CODE REVIEW

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html [hunk 1]
```diff
@@ -13,120 +13,126 @@
...
-        *matRowDef="let row; columns: displayedColumns;"
-        matRipple
-        class="element-row"
-        [appJobResultsDetailRow]="row"
-        [appJobResultsDetailRowTpl]="tpl"
-        (toggleChange)="toggleChange($event)">
-    </tr>
-  </table>
-</mat-card>
+<mat-error *ngIf="tableData.length === 0; else results">
```
Comment (AlbaLopez) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html:17]: I like that you use ngIf else structure 👍

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html [hunk 2]
```diff
@@ -171,3 +177,4 @@ <h3 class="inline">Results</h3>
     <td colspan="6"></td>
   </tr>
 </ng-template>
+
```
Comment (AlbaLopez) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: delete this blank line

File: src/app/jobs/components/job-detail/job-detail.component.html [hunk 3]
```diff
@@ -87,9 +87,12 @@ <h2>
...
         [tableData]="tableData"
+        [productName] = "job.product_name"
+        [organism$]="organism"
         [reactions]="reactionsData"></app-job-results-table>
     </div>
   </mat-card>
 </div>
 
 <ng-template #loadingTemplate>Loading...</ng-template>
+
```
Comment (AlbaLopez) [inline src/app/jobs/components/job-detail/job-detail.component.html]: delete blank line

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html [hunk 4]
```diff
@@ -13,120 +13,126 @@
...
-        matRipple
-        class="element-row"
-        [appJobResultsDetailRow]="row"
-        [appJobResultsDetailRowTpl]="tpl"
-        (toggleChange)="toggleChange($event)">
-    </tr>
-  </table>
-</mat-card>
+<mat-error *ngIf="tableData.length === 0; else results">
+    There are no pathways to <strong>{{productName}}</strong> in <strong>{{organism$ | async}}</strong>.
```
Comment (AlbaLopez) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: The product is not inside of the job details??

Comment (AlbaLopez) [inline src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: Why organism$ is an observable?

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html (hunk 1 above)
Comment (annasirunian) [reply src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html:17]: Thanks! :)

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html (hunk 2 above)
Comment (annasirunian) [reply src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: Done

File: src/app/jobs/components/job-detail/job-detail.component.html (hunk 3 above)
Comment (annasirunian) [reply src/app/jobs/components/job-detail/job-detail.component.html]: Done

File: src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html (hunk 4 above)
Comment (annasirunian) [reply src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.html]: I changed the code as it was before with getting organism name by its id.

**Review by AlbaLopez — APPROVED** [review]

**Pull request merged**